benchmark/metrics/cputime.py

```diff
@@ -36,19 +36,28 @@
     b'-C',
 ]
 
+# A string that not a valid process name.  Any name with a space is good
+# because metric does not support process names containing spaces.
+WALLCLOCK_LABEL = '-- WALL --'
+
 
 class CPUParser(LineOnlyReceiver):
     """
     Handler for the output lines returned from the cpu time command.
     """
 
-    def __init__(self):
+    def __init__(self, reactor):
+        self.reactor = reactor
         self.result = {}
```

Comment:
:question: I'd suggest prefixing both of these attributes with `_` since they aren't (AIUI) meant to be accessed externally.